docs/reference/modules/_template.md

```diff
@@ -27,7 +33,7 @@ The following methods belong to this module and may be useful in project-level c
 
 Because this module has an alias, you can call these from another module from the alias path. For example, `self.apos.[the alias].inferIdLocaleAndMode()`.
 
-### `inferIdLocaleAndMode(req, _id)`
+### `async inferIdLocaleAndMode(req, _id)`
```

Comment:
Adding this as a reminder to identify async methods this way. I realize this particular method isn't async, but this is simply the template.